Zend/tests/generators/gh9750-008.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Bug GH-9750 008 (Generator memory leak when interrupted during argument evaluation)
+--SKIPIF--
+<?php
+if (version_compare(PHP_VERSION, '8.1', '<')) { die('skip Broken on PHP < 8.1'); }
```

Comment:
8.0 calls `D::__destructor`: https://3v4l.org/32Ku7
8.1 does not because the constructor has not been called